### PR TITLE
Add skip feature

### PR DESCRIPTION
**Pull request by yo1dog**
Per #16

- Adds ability to skip all device mounting and DOL loading by holding d-left or the reset button.
- Delays exit until reset button is also released (to prevent restarting onboard IPL when button is released).
- Adds diagnostic messages instructing user to release button(s) when exit is delayed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,9 @@ CLI files are also supported.
 
 If the selected file cannot be loaded, iplboot will reboot to the original IPL (GameCube intro and menu) instead.
 
-For example, this configuration would boot straight into Swiss by default, or GBI if you held B, or the original GameCube intro if you held any other button:
+Holding D-Pad Left or the reset button will skip iplboot functionality and reboot straight into the onboard IPL.
+
+For example, this configuration would boot straight into Swiss by default, or GBI if you held B, or the original GameCube intro if you held D-Pad Left:
```
Comment (9ary) [inline README.md:33]: I think this slipped in accidentally.

Comment (yo1dog) [reply README.md:33]: What slipped in?

Comment (9ary) [reply README.md:33]: Nevermind, I was confused.

**Comment by yo1dog** [comment]
Moved removal of d-pad shortcut support from #19 to here.

File: source/main.c [hunk 2]
```diff
@@ -355,6 +364,17 @@ int main()
...
+    {
+        // Since we've disabled the Qoob, we wil reboot to the Nintendo IPL
+        kprintf("Skipped. Rebooting into original IPL...\n");
+        delay_exit();
+        return 0;
+    }
+
+    char *paths[2];
+    int num_paths = 0;
+
```
Comment (9ary) [inline source/main.c:374]: Ah right, this is what slipped in heh. I'm cherry-picking your changes manually so it's fine.

**Pull request closed without merging**